Readme.md

```diff
@@ -17,11 +16,11 @@ The example includes solutions for both Blazor Server and Blazor WebAssembly hos
 
 Follow the steps below to implement a Theme Switcher in your application:
 
-1. Use a [DevExpress Project Template](https://docs.devexpress.com/Blazor/401057/get-started) to create a new Blazor Server or Blazor WebAssembly application.
+1. Use a [DevExpress Project Template](https://docs.devexpress.com/Blazor/401057/get-started) to create a new Blazor Server or Blazor WebAssembly application. The newly created project references the **DevExpress.Blazor** NuGet package that contains DevExpress Blazor UI components and links a package with built-in DevExpress themes.

```

Comment:
Can we name the theme package name in the parentheses?

Reply:
```suggestion
1. Use a [DevExpress Project Template](https://docs.devexpress.com/Blazor/401057/get-started) to create a new Blazor Server or Blazor WebAssembly application. The newly created project references the **DevExpress.Blazor** NuGet package that contains DevExpress Blazor UI components and links a package with built-in DevExpress themes (the **DevExpress.Blazor.Themes** package).

```